src/components/SchemaTable/index.js

```diff
@@ -137,12 +156,20 @@ export default class SchemaTable extends React.PureComponent {
       <Container
         backgroundColor={this.props.headerBackgroundColor}
         maxHeight={this.props.maxHeight}
-        schema={schema}>
-        <Table
-          responsive
-          className={styles.parentTable}>
-          {this.schemaTable(schema, null, null, schema.$id ? schema.$id : schema.id)}
-        </Table>
+        schema={schema}
+        jsonView={this.state.jsonView}
+        onClick={this.handleOnClick}>
```

Comment:
The name `onClick` seems a little too general here -- someday we might have multiple things to click on.  Also, why does SchemaTable need to be worried about this?  I suspect Container could just render either `this.props.children` or the JSON, right?  Then SchemaTable wouldn't have to change at all.

Reply:
yes. this makes sense. let me make this change right away